cypress/integration/happy-path.spec.js

```diff
@@ -0,0 +1,91 @@
+describe('Happy path testing', function() {
+  it('Should complete happy path successfully from scope diagnosis to confirmation page', function () {
+
+    cy.visit("https://lga-1543-install-and-integra.staging.checklegalaid.service.gov.uk/start")
+    
+    cy.contains('Choose the area you most need help with')
+    cy.savePage('homepage')
+    cy.contains('Debt').click()
+
+    cy.contains('Choose the option that best describes your debt problem')
+    cy.savePage('debt-options')
+    cy.get(':nth-child(1) > .cla-scope-options-list-item-link').click()
+
+    cy.contains('Legal aid is available for this type of problem')
+    cy.savePage('scope-outcome')
+    cy.contains('Check if you qualify financially').click()
+
+    const someNumber = 3245;
```

Comment:
worth renaming this variable to something a little bit more meaningful like 'generalMonetaryValue' or something like that to make it read better.